smart_open/utils.py

```diff
@@ -223,3 +223,10 @@ def __exit__(self, *args, **kwargs):
 
     def __next__(self):
         return self.__wrapped__.__next__()
+
+    def close(self):
+        try:
+            self.__wrapped__.close()
```

Comment:
```suggestion
            return super().close()
```
analogous to `__exit__` above (maybe some wrapped classes might return a boolean or something)

Reply:
Hi @ddelange ,

I am trying the `super().close()` call but the unit test returns an error:
```python
    def close(self):
        try:
            # return self.__wrapped__.close()
>           return super().close()
E           AttributeError: 'super' object has no attribute 'close'

../utils.py:230: AttributeError
```

I noticed that you have `__wrapper__.next()` in the class method above. Probably because `wrapt.ObjectProxy` does not implement it; similar scenario with the class method `close()`).

I am implementing the return part in case `close()` return something in different implementation. Thanks!!

Reply:
alright, thanks for checking!